src/Employer.Web/Controllers/DashboardController.cs

```diff
@@ -1,22 +1,78 @@
+using Esfa.Recruit.Employer.Web.Configuration;
 using Esfa.Recruit.Employer.Web.Configuration.Routes;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Esfa.Recruit.Employer.Web.Extensions;
 
 namespace Esfa.Recruit.Employer.Web.Controllers
 {
     public class DashboardController : Controller
     {
         ILogger<DashboardController> _logger;
+        private readonly IOptions<AuthenticationConfiguration> _authConfig;
+        private readonly IOptions<ExternalLinksConfiguration> _extLinksConfig;
 
-        public DashboardController(ILogger<DashboardController> logger)
+        public DashboardController(ILogger<DashboardController> logger, IOptions<AuthenticationConfiguration> authConfig, IOptions<ExternalLinksConfiguration> extLinksConfig)
         {
             _logger = logger;
+            _authConfig = authConfig;
+            _extLinksConfig = extLinksConfig;
         }
 
         [HttpGet, Route("accounts/{employerAccountId}/dashboard", Name = RouteNames.Dashboard_Index_Get)]
         public IActionResult Index()
         {
             return View();
         }
+
+        [HttpGet("accounts/{employerAccountId}/dashboard/change-password", Name = RouteNames.Dashboard_ChangePassword)]
```

Comment:
I would prefer that. But you said before that you wanted all routes to be specific and only on the actual action method. This was partly because I thought that the route naming for `HttpPost` actions was/is redundant. Happy to change it.